Numerge.Gui/.gitignore

```diff
@@ -0,0 +1,337 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Looks like this `.gitignore` file was included here accidentally, while another one exists in the repository root :smile_cat: